### DISCUSSION TITLE
Step up/down  Start % Stop % Question

### DISCUSSION THREAD

**Discussion by alex3xelax** [Rem0o/FanControl.Releases#871] [Q&A]
![Annotazione 2022-06-13 194527](https://user-images.githubusercontent.com/88495460/173415355-b71e9e0d-1a51-4b5e-9424-b279989cd77d.png)

this screen rappresent my Cpu fan.
i use automatic start % & stop % detection, i got 18% , 0%.
- i can use these values or i should put 0% and 0% as suggested in other threads ?

- The step up, step down rappresent the offset of the fan when rump up or rump down ?

thanks

**Comment by Rem0o** [comment]
The values seems right. The fan probably nevers stops, hence 0%.

Step down and step up are the maximum rate at which the fan can change speed. That's for you to tweak.

**Reply by alex3xelax** [reply]
so, if  i have the cpu fan **always on**
![Annotazione 2022-06-13 194527](https://user-images.githubusercontent.com/88495460/173449344-fa3a6491-69d1-4ab2-8bff-b8d714c8bf10.png)
![Annotazione 2022-06-13 233503](https://user-images.githubusercontent.com/88495460/173449356-74528029-be4a-4fe2-81b0-04ec41699451.png)

Should i set manually **start% and stop% = 0** ?

In my rig only the gpu use 0rpm feauture.

**Reply by Rem0o** [reply]
Yup that’s perfectly fine.

**Comment by alex3xelax** [comment]
**start% and stop% = 0**  what exactly mean ?

thanks

**Reply by bluespresso** [reply]
Start % is only relevant if stop% > 0%.
So let`s assume Stop% = 10%
You have to distinguish between two cases:
1)	Your fan is spinning. If your fan curve requests less power than Stop% is set to, then your fan will stop as soon as it reaches 10%.
2)	Case 2. Your fan is currently not spinning. When your fan curve requests power to be more than Stop% then your fan will start spinning initially with the power of Start% for a short while as a kick start. Then it will set whatever power the fan curve is requesting.

Make sure to set Stop% not less than your fan can handle with constant power. If you set it too low it will continuously ramp up and down as the program prevents it to fully stop while powered. You can test this very well with the manual control feature and monitor fan rpm.

If you want to implement a proper working start/stop setting you need to follow a few steps.
The temperature your fans start spinning (e.g. 60°C) needs to be higher than the temperature your fans stop spinning (e.g. 50°C). Your cooler needs to have a least some passive capability otherwise the necessary gap would be too large.
You can accomplish this by combining a graph fan curve with a target fan curve.
Have a read here:
https://github.com/Rem0o/FanControl.Releases/discussions/862
